crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -651,19 +648,19 @@ Added a new build of nrf_cc3xx_platform libraries for nRF9160, nRF52840, and nRF
 nrf_cc3xx_platform - 0.9.8
 **************************
 
-New version of the library with improvements and bug fixes:
+New version of the library with the following improvements and bug fixes:
 
-* Decreased stack usage for PRNG using `CTR_DRBG`.
-* Fixed issue with `CTR_DRBG` usage on the platform library when multiple backends are enabled in nordic security.
+* Decreased stack usage for PRNG using ``CTR_DRBG``.
+* Fixed issue with ``CTR_DRBG`` usage on the platform library when multiple backends are enabled in Nordic security.
```

Comment:
The term we have been trying to establish is nrf_security. "Nordic security" is maybe a bit vague in the context

Reply:
'Nordic security' would, unless it's an established term, violate the rule to never shorten the brand name. 

Is `nrf_security` a file or option, to be formatted as such? Or is it a general concept we should format as nRF security?

Reply:
What do you think @frkv?

Reply:
Nordic Security Module (nrf_security)

Reply:
Changed as suggested by Mia.